mmv1/products/cgc/terraform.yaml

```
@@ -72,6 +72,22 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "instance_virtual_display"
         vars:
           instance_virtual_display: "instance-virtual-display"
+ 
+    # Eventarc
+    ### Eventarc Workflows quickstart
+      - !ruby/object:Provider::Terraform::Examples
+        name: "eventarc-workflows"
```

```suggestion
        name: "eventarc_workflows"
```
We will want to use an underscore instead of the dash here, and we'll need to update the associated filename as well. Thanks!